apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+// <svg class="c-pieIcon c-pieIcon--x foo bar" stroke-width="1" color="red" ...><line ... /><line ... /></svg>
+```
+
+{% notification {
+  type: "information",
+  message: "For more information about the APIs in this package, please take a look at the [API reference](https://github.com/justeattakeaway/pie/tree/main/packages/tools/pie-icons#api-reference) in the project README."
+} %}
+
+
+---
+
+## React
+
+This package generates an icon set for React applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into React components.
+
+The icons are bundled in CommonJS and ES Modules, to be more easily adopted for modern React Applications.
+
+
+### Usage
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons-react --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons-react
+```
+
+#### Import into your project
+Simply import the icons you need into your project:
+
+```tsx
+import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-react";
+
+export default function App() {
+  return (
+    <div className="App">
+      <IconCalendar />
+      <IconAlertTriangleLarge fill={PIE_ALIAS_COLOR_TOKEN} />
+    </div>
+  );
+}
+```
+
+{% notification {
+  type: "information",
+  message: "For more information, please refer to the package [README](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-icons-react/README.md)."
+} %}
+
+---
+
+## Vue
+
+This package generates an icon set for Vue.js applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into single file components that can be imported into Vue applications.
+
+{% notification {
+  type: "warning",
+  message: "The package is fully compatible with Vue 2. To use these icons in Vue 3, you may need to take some extra steps when building your application. Please reach out to **#help-designsystem** for assistance."
+} %}
+
+### Usage
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons-vue --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons-vue
+```
+
+#### Import into your project
+Simply import the icons you need into your project:
+
+```html
+  <template>
+      <icon-calendar />
+      <icon-alert-triangle-large fill={PIE_ALIAS_COLOR_TOKEN} />
+  </template>
+
+  <script>
+  import { IconCalendar, IconAlertTriangleLarge } from '@justeattakeaway/pie-icons-vue';
+
+  export default {
+      components: {
+          IconCalendar,
+          IconAlertTriangleLarge
+      }
+  };
+  </script>
+  ```
+
+{% notification {
+  type: "information",
+  message: "For more information, please refer to the package [README](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-icons-vue/README.md)."
+} %}
+
+---
+## Web Components
```

Comment:
Same here:

```md
## `pie-icons-webc`
```

or 
```md
## Web Components (`pie-icons-webc`)
```

Reply:
Sure - it doesn't look great using back ticks in the header so I'll go with the latter format but omit the back ticks :)